### DISCUSSION TITLE
Making addons like Tampermonkey work in the sidepanel pages

### DISCUSSION THREAD

**Discussion by MannixMD** [zen-browser/desktop#1793] [Ideas]
As of right now the scripts you can add/install in addons like Tampermonkey do not work in the pages open in sidepanel. Other addons that let's you modify webpages like Stylus do work in sidepanel pages. Is it possible to make the scripts work too??

**Comment by MannixMD** [comment]
I can happily report that the issue preventing scripts from working in sidepanel pages was fixed in 1.0.1-a.6

**Comment by xologram** [comment]
I am on 1.0.2-b.5 (macOS) and tampermonkey scripts do not work for me in web panel. Am I missing something?
